focus-android/app/metrics.yaml

```diff
@@ -762,11 +791,12 @@ preferences:
     data_reviews:
       - https://github.com/mozilla-mobile/focus-android/pull/5526
       - https://github.com/mozilla-mobile/focus-android/pull/7418#issuecomment-1195518264
+      - https://github.com/mozilla-mobile/firefox-android/pull/3320
     data_sensitivity:
       - interaction
     notification_emails:
       - android-probes@mozilla.com
-    expires: 119
+    expires: 123
```

Comment:
I don't see `user_theme` in the Request for Data Collection Renewal - should it be?

Reply:
Good catch.  Missed it when creating the request.  I've updated the 3 months request to include the `user_theme` probe.  Thanks!